examples/angular/src/pages/ui/components/authenticator/use-authenticator/use-authenticator.component.html

```diff
@@ -0,0 +1,27 @@
+<amplify-authenticator>
```

Comment:
Here's the Angular `useAuthenticator` example that demos "tabless" authenticator. Tabs are hidden through css here: 

https://github.com/aws-amplify/amplify-ui/blob/c32ef37a641de52e3d54a3fe7f9af5ae2c6aa1db/examples/angular/src/styles.scss#L15-L17

Screenshot:
![Screen Shot 2021-11-11 at 8 47 39 PM](https://user-images.githubusercontent.com/43682783/141411367-8b57eaf6-89ee-4da8-86e3-cf02eed4a284.png)

![image](https://user-images.githubusercontent.com/43682783/141411459-5c4c7e8a-a53f-4fb5-a344-35203e42084f.png)

Reply:
@ErikCH @ericclemmons Please add matching examples to this PR, thanks! 🙏

Reply:
Moving this to a [Headless UI](https://github.com/aws-amplify/amplify-ui/milestone/2) milestone to do after GA, since this is an advanced use-case.